### ISSUE TITLE
Can't change background color of the sticky header

### ISSUE THREAD

**Issue by scottsgcho** [jaysoo/react-native-parallax-scroll-view#66]
Hi, is there a way to change the background color for the sticky header?

The default color for the header is black, and I can't override it by nesting a view because the view slides up from the bottom.

Thank you for the support.

**Comment by bispul** [comment]
exactly, whatever style we apply to the view of sticky header it slides up from the bottom. 
@scottsgcho Did you solve that issue?

**Comment by mickadoua** [comment]
maybe
` <ParallaxScrollView  backgroundColor="#FF0"   >` ?

**Comment by dengue8830** [comment]
duplicated from #51 `backgroundColor` is the answer, do not mistake with `headerBackgroundColor`
